File: CityGuess/app/src/main/java/pl/edu/pwr/lab46/i236764/MyApplication.java
package pl.edu.pwr.lab46.i236764;

import android.app.Application;

import pl.edu.pwr.lab46.i236764.QuizApp;

public class MyApplication extends Application {
    @Override
    public void onCreate()
    {
        super.onCreate();
        QuizApp.init(this);
    }

}

File: CityGuess/app/src/main/java/pl/edu/pwr/cityguess/MyApplication.kt
package pl.edu.pwr.cityguess

import android.app.Application
import pl.edu.pwr.cityguess.DatabaseHelper
import pl.edu.pwr.cityguess.HighScoreActivity
import pl.edu.pwr.cityguess.KvizApp
import pl.edu.pwr.cityguess.PreferencesHelper
import pl.edu.pwr.cityguess.QuestionItem
import pl.edu.pwr.cityguess.ScoreItem

/**
 * Created by stevyhacker on 24.11.14..
 */
class MyApplication : Application() {
    override fun onCreate() {
        super.onCreate()
        KvizApp.init(this)
    }
}